src/render/vitagxm/SDL_render_vita_gxm_tools.c

```diff
@@ -1178,6 +1178,77 @@ gxm_texture_set_filters(gxm_texture *texture, SceGxmTextureFilter min_filter, Sc
     sceGxmTextureSetMagFilter(&texture->gxm_tex, mag_filter);
 }
 
+static unsigned int back_buffer_index_for_common_dialog = 0;
+static unsigned int front_buffer_index_for_common_dialog = 0;
+struct
+{
+    VITA_GXM_DisplayData displayData;
+    SceGxmSyncObject* sync;
+    SceGxmColorSurface surf;
+    SceUID uid;
+} buffer_for_common_dialog[VITA_GXM_BUFFERS];
+
+void gxm_minimal_init_for_common_dialog(void)
+{
+    SceGxmInitializeParams initializeParams;
+    SDL_zero(initializeParams);
+    initializeParams.flags                          = 0;
+    initializeParams.displayQueueMaxPendingCount    = VITA_GXM_PENDING_SWAPS;
+    initializeParams.displayQueueCallback           = display_callback;
+    initializeParams.displayQueueCallbackDataSize   = sizeof(VITA_GXM_DisplayData);
+    initializeParams.parameterBufferSize            = SCE_GXM_DEFAULT_PARAMETER_BUFFER_SIZE;
+    sceGxmInitialize(&initializeParams);
+}
+
+void gxm_minimal_term_for_common_dialog(void)
+{
+    sceDisplayGetFrameBuf(NULL, SCE_DISPLAY_SETBUF_NEXTFRAME);
+    sceGxmTerminate();
+}
+
+void gxm_init_for_common_dialog(void)
+{
+    for (int i = 0; i < VITA_GXM_BUFFERS; i += 1)
+    {
+        buffer_for_common_dialog[i].displayData.wait_vblank = true;
```

Comment:
This line is erroring for me during build using VitaSDK:

```
SDL_render_vita_gxm_tools.c: In function 'gxm_init_for_common_dialog':
SDL_render_vita_gxm_tools.c:1213:63: error: 'true' undeclared (first use in this function)
 1213 |         buffer_for_common_dialog[i].displayData.wait_vblank = true;
```
Suggested fix:

```suggestion
        buffer_for_common_dialog[i].displayData.wait_vblank = SDL_TRUE;
```